plugin/storage/memory/memory_test.go

```diff
@@ -53,6 +54,32 @@ var testingSpan = &model.Span{
 	StartTime: time.Unix(300, 0).UTC(),
 }
 
+var traceID2 = model.NewTraceID(2, 3)
+
+var testingSpan2 = &model.Span{
```

Comment:
nit: turn `testingSpan` into a function that takes a few args distinguishing the traces (like trace id)